client/app/account/user/security/users/group.service.js

```diff
@@ -0,0 +1,23 @@
+"use strict";
+angular.module("UserAccount.services").service("UseraccountGroupsService", class UseraccountGroupsService {
+
+    constructor (OvhHttp) {
+        this.ovhHttp = OvhHttp;
+    }
+
+    getGroups () {
+        return this.ovhHttp.get("/me/identity/group", {
+            rootPath: "apiv6"
+        });
+    }
+
+    getGroup (groupName) {
```

Comment:
You can name your param as `group` and then using object shorthand syntax like so:

```js
urlParams: {
    group
}
```

Reply:
Done